### DISCUSSION TITLE
Question about the quantification of the PG.Quantity value

### DISCUSSION THREAD

**Discussion by diabeginner01** [vdemichev/DiaNN#537] [Q&A] [answered]
I have noticed that if I run my DIA files with pretty much the default settings, I end up with results where the non-normalized PG quantity ends up being reported only by one of the precursor's intensity, even though the PG has multiple of precursors. It seems that this is based on the most intense precursor or whether it has the highest Quantity.Quality value. What is the reasoning behind this using only one precursor to provide the entire PG's quantity? (Q1) Is the PG.MaxLFQ value then calculated based on all the precursors of a PG or also based on a single (most intense, best quality) precursor (?), as I've understood that for MaxLFQ all precursors/peptides are taken into account (Q2). 
What is even more puzzling is that for some proteins the most intense / best quality precursor is taken but the peptide is not proteotypic...while there are other peptides available that are proteotypic but are just less intense than the one picked. Why would DIA-NN take the intensity of a non-proteotypic peptide to report the PG's quantity, instead of taking the one that is proteotypic? Is this related to using the highly heuristic protein inference? (Q3)

One example of a PG where this is observed (precursor SYSC(UniMod:4)QVTHEGSTVEK3 is the one that gives the whole PG its quantity value):

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/sergo/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/sergo/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Protein.Group | Genes | PG.Quantity | PG.Normalised | PG.MaxLFQ | Precursor.Id | Precursor.Charge | Q.Value | PEP | Global.Q.Value | Protein.Q.Value | PG.Q.Value | Proteotypic | Precursor.Quantity | Precursor.Normalised | Precursor.Translated | Ms1.Translated | Quantity.Quality
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
B9A064 | IGLL5 | 4.85E+05 | 1.32E+06 | 9.94E+05 | AGVETTKPSK2 | 2 | 0.000284 | 0.000398 | 4.60E-05 | 0.0123267 | 0.00220751 | 0 | 2.58E+04 | 6.80E+04 | 2.98E+04 | 1.11E+05 | 0.847033
B9A064 | IGLL5 | 4.85E+05 | 1.32E+06 | 9.94E+05 | SYSC(UniMod:4)QVTHEGSTVEK2 | 2 | 0.000284 | 0.000378 | 5.29E-05 | 0.0123267 | 0.00220751 | 0 | 1.28E+05 | 3.53E+05 | 1.55E+05 | 6.98E+05 | 0.951326
B9A064 | IGLL5 | 4.85E+05 | 1.32E+06 | 9.94E+05 | SYSC(UniMod:4)QVTHEGSTVEK3 | 3 | 0.000284 | 0.004108 | 0.000484261 | 0.0123267 | 0.00220751 | 0 | 4.85E+05 | 1.32E+06 | 5.81E+05 | 1.61E+06 | 0.977647
B9A064 | IGLL5 | 4.85E+05 | 1.32E+06 | 9.94E+05 | TVAPTEC(UniMod:4)S1 | 1 | 0.000763 | 0.102637 | 0.000484261 | 0.0123267 | 0.00220751 | 0 | 4.51E+04 | 1.23E+05 | 5.41E+04 | 0.00E+00 | 0.935244
B9A064 | IGLL5 | 4.85E+05 | 1.32E+06 | 9.94E+05 | VTVLGQPK2 | 2 | 0.000284 | 0.00048 | 0.000484261 | 0.0123267 | 0.00220751 | 1 | 3.65E+04 | 1.15E+05 | 5.04E+04 | 2.65E+05 | 0.938049
B9A064 | IGLL5 | 4.85E+05 | 1.32E+06 | 9.94E+05 | YAASSYLSLTPEQWK2 | 2 | 0.000187 | 0.000187 | 0.000484261 | 0.0123267 | 0.00220751 | 0 | 1.70E+05 | 6.07E+05 | 2.66E+05 | 1.73E+06 | 0.948973

</body>

</html>

**Comment by vdemichev** [comment, marked as answer]
It's Top 1 method, wors best. But we recommend using MaxLFQ quantities instead - calculated based on all precursors. There's special proteotypic-based quantity: Genes.MaxLFQ.Unique. Also, you can calculate protein quantities yourself in whatever way you like using 'iq' or 'diann' R package.

**Reply by JianAtSeer** [reply]
Hi Vadim:
IJust to clarify, so the default method for the protein quant is top 1 in the PG.Quant or PG.Normalised column?  I thought it was top3 from reading the paper (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6949130/).  Has this been always the case? Or this become the default since some specific release? Thanks a lot.

**Comment by diabeginner01** [comment]
OK thanks for the clarification!
